x-pack/plugins/observability/public/pages/slos/components/slo_list_items.tsx

```diff
@@ -29,13 +30,19 @@ export function SloListItems({ sloList, loading, error }: Props) {
   const { isLoading: historicalSummaryLoading, sloHistoricalSummaryResponse } =
     useFetchHistoricalSummary({ sloIds });
 
+  const { mutate: deleteSlo } = useDeleteSlo();
+
   if (!loading && !error && sloList.length === 0) {
     return <SloListEmpty />;
   }
   if (!loading && error) {
     return <SloListError />;
   }
 
+  const handleDelete = (slo: SLOWithSummaryResponse) => {
```

Comment:
By handling the deleting in this parent component, the React error when deleting an item via the SLO List goes away.